pkg/util/bindings/knative_ref.go

```diff
@@ -150,6 +150,17 @@ func (k V1alpha1KnativeRefBindingProvider) Translate(ctx V1alpha1BindingContext,
 		return nil, nil
 	}
 
+	if ok, err := isKnownKnativeResource(e.Ref); !ok {
```

Comment:
This change is done on the `V1alpha1KnativeRefBindingProvider` only. I think we need this for the new `KnativeRefBindingProvider` as well (the former is the one used for deprecated `KameletBinding`).

Reply:
Forget my previous comment. I've just realized this one amended the previous work.